nss_cache/sources/httpsource_test.py

```diff
@@ -229,7 +232,7 @@ def testRetryOnErrorCodeResponse(self):
                           since=None)
 
 
-class TestPasswdUpdateGetter(unittest.TestCase):
+class TestPasswdUpdateGetter(mox.MoxTestBase):
```

Comment:
These tests don't use mox though, do they?

Reply:
No, they don’t. I’ll revert that change.